src/core/server/core_app/bundle_routes/file_hash_cache.ts

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-import LruCache from 'lru-cache';
```

Comment:
All the changes from here and below were approved in https://github.com/elastic/kibana/pull/206225